docs/contributor/01-00-installation.md

```diff
@@ -0,0 +1,241 @@
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
+- [Docker](https://www.docker.com)
+- [Kyma CLI](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/01-install-kyma-CLI)
+
+## Install Istio Operator manually
+
+1. Clone the project.
+
+```bash
+git clone https://github.com/kyma-project/istio.git && cd istio
+```
+
+2. Set the Istio Operator image name.
+
+```bash
+export IMG=istio-operator:0.0.1
+export K3D_CLUSTER_NAME=kyma
+```
+
+3. Provision k3d cluster.
+
+```bash
+kyma provision k3d
+```
+>**TIP:** To verify the correctness of the project, build it using the `make build` command.
+
+4. Build the image.
+
+```bash
+make docker-build
+```
+
+5. Push the image to the registry.
+
+<div tabs name="Push image" group="istio-operator-installation">
+  <details>
+  <summary label="k3d">
+  k3d
+  </summary>
+
+   ```bash
+   k3d image import $IMG -c $K3D_CLUSTER_NAME
+   ```
+
+  </details>
+  <details>
+  <summary label="Docker registry">
+  Globally available Docker registry
+  </summary>
+
+   ```bash
+   make docker-push
+   ```
+
+  </details>
+</div>
+
+6. Deploy.
+
+```bash
+make deploy
+```
+
+### Use Istio Operator to install or uninstall Istio
+
+- Install Istio in your cluster.
+
+```bash
+kubectl apply -f config/samples/operator_v1alpha1_istio.yaml
+```
+
+- Delete Istio from your cluster.
+
+```bash
+kubectl delete -f config/samples/operator_v1alpha1_istio.yaml
+```
+
+## Install Istio in modular Kyma on a local k3d cluster
+
+1. Set up a local k3d cluster and a local Docker registry.
+
+```bash
+k3d cluster create kyma --registry-create k3d-kyma-registry:0.0.0.0:5001
+```
+
+2. Add the `etc/hosts` entry to register the local Docker registry under the name `k3d-kyma-registry`.
+
+```bash
+127.0.0.1 registry.localhost
+```
+
+3. Export environment variables pointing to the module and the module's image registries.
+
+```bash
+export IMG_REGISTRY=registry.localhost:5001/unsigned/operator-images
+export MODULE_REGISTRY=registry.localhost:5001/unsigned
+```
+
+4. Build the Istio module.
+
+```bash
+make module-build
+```
+
+This command builds an OCI image for the Istio module and pushes it to the registry and path, as defined in `MODULE_REGISTRY`.
+
+5. Build Istio Operator's image.
+
+```bash
+make module-image
+```
+
+This command builds a Docker image for Istio Operator and pushes it to the registry and path, as defined in `IMG_REGISTRY`.
+
+6. Verify that the Istio module's image and Istio Operator's image are pushed to the local registry. Run:
+
+```bash
+curl registry.localhost:5001/v2/_catalog
+```
+If the images were pushed successfully, you get the following output:
+
+```json
+{
+    "repositories": [
+        "unsigned/component-descriptors/kyma.project.io/module/istio",
+        "unsigned/operator-images/istio-operator"
+    ]
+}
+```
+
+7. Inspect the generated module template. Change the existing repository context in `spec.descriptor.component`:
```

Comment:
"module template" or "ModuleTemplate" meaning the CR?

Reply:
It should be ModuleTemplate. I corrected it